File: 프로그래머스/lv1/68644. 두 개 뽑아서 더하기/두 개 뽑아서 더하기.py
def solution(numbers):
    answer = []
    p_list = []
    p_set = ()
    
    n = len(numbers)
    
    for i in range(n-1):
        for j in range(i + 1, n):
            p_list.append(numbers[i] + numbers[j])
            
    p_set = set(p_list)
    p_list = list(p_set)
    
    answer = sorted(p_list)   
            
    return answer

File: 프로그래머스/1/68644. 두 개 뽑아서 더하기/두 개 뽑아서 더하기.js
function solution(numbers) {
  const list = [];

  for (let i = 0; i < numbers.length; i++) {
    for (let j = i + 1; j < numbers.length; j++) {
      list.push(numbers[i] + numbers[j]);
    }
  }

  return [...new Set(list)].sort((a, b) => a - b);
}
